ada_feeding/scripts/ada_planning_scene.py

```diff
@@ -64,7 +64,10 @@ def load_parameters(self) -> None:
             descriptor=ParameterDescriptor(
                 name="assets_dir",
                 type=ParameterType.PARAMETER_STRING,
-                description="The absolute path to the directory to find the assets (e.g., STL mesh files).",
+                description=(
+                    "The absolute path to the directory to find the assets "
+                    "(e.g., STL mesh files)."
```

Comment:
JW for formatting did you use Black Formatter (`python3 -m black .`) or something else? And if black, did you add any flags to it?

We should just ensure that we're using the same formatter to avoid successive PRs making/reverting style changes. I use `python3 -m black .`

Reply:
No flags w/ black. This was a compromise between black and pylint that was complaining about the long lines.